packages/@aws-cdk/aws-stepfunctions/test/task-base.test.ts

```diff
@@ -104,6 +105,36 @@ describe('Task base', () => {
     });
   });
 
+  test('States.HeartbeatTimeout is added correctly', () => {
```

Comment:
tbh this test is unneeded – we're just adding a constant string. I'd delete unless you feel strongly